zenodo/modules/pages/templates/zenodo_pages/contact_form.html

```diff
@@ -93,11 +236,96 @@
     var dropdown = document.getElementById('issue_category');
     var current_value = dropdown.options[dropdown.selectedIndex].value;
 
-    for(each in data){
-      if(data[each][0]==current_value){
+    for(each in data)
```

Comment:
You can easily transform the `data` to a dict/object before and access the text by index